nemo/core/classes/modelPT.py

```diff
@@ -307,10 +307,7 @@ def _update_artifact_paths(self, path2yaml_file):
         if self.artifacts is not None and len(self.artifacts) > 0:
             conf = OmegaConf.load(path2yaml_file)
             for conf_path, item in self.artifacts.items():
-                if item.hashed_path is None:
-                    conf.update_node(conf_path, item.path)
-                else:
-                    conf.update_node(conf_path, item.hashed_path)
+                OmegaConf.update(conf, conf_path, item.path)
             with open(path2yaml_file, 'w') as fout:
```

Comment:
This is incorrect - needs to have two paths as before just with updated syntax. I think I missed the else path in the previous patch.

Reply:
Good catch I'll update it.